nginx/default.conf

```diff
@@ -16,6 +16,9 @@ server {
   ssl on;
   ssl_certificate /etc/letsencrypt/live/logging.dockstore.org/fullchain.pem;
   ssl_certificate_key /etc/letsencrypt/live/logging.dockstore.org/privkey.pem;
+  ssl_protocols TLSv1.2 TLSv1.3;
+  ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
```

Comment:
This matches the cipher string from https://ssl-config.mozilla.org/, but not from https://linux-audit.com/protect-against-the-beast-attack-in-nginx/. Is that on purpose?

Reply:
Yes, the first article suggested using Mozilla's list to get the most up-to-date.